src/style.css

```diff
@@ -1,96 +1,84 @@
-@import url('https://fonts.googleapis.com/css2?family=Rubik:wght@300&display=swap');
-
 :root {
-    --purple: #3A233E;
-    --orange: #bf5701;
-    --light-orange: #e6a570;
+  --light-orange: #e6a570;
+  --orange: #bf5701;
+  --bg: #3A233E;
+  --text: #fff;
+}
+
+html {
+  box-sizing: border-box;
 }
 
 body {
-    background-color: var(--purple);
-    font-family: "Montserrat", sans-serif;
+  background-color: var(--bg);
+  color: var(--text);
+  font-family: "Montserrat", sans-serif;
 }
 
 .logo {
-    display: block;
-    margin: auto;
-    fill: var(--orange);
+  display: block;
+  margin: auto;
 }
 
 .container {
-    margin: auto;
-    margin-top: 5%;
-    width: 50%;
-    padding: 10px;
-    text-align: center;
+  margin: 5% auto 0;
+  padding: 10px;
+  text-align: center;
+  width: 50%;
 }
 
 h1 {
-    color: white;
-    font-weight: 500;
-    font-size: 3rem;
+  font-size: 3rem;
+  font-weight: 500;
 }
 
 h2 {
-    color: white;
-    font-weight: 300;
+  font-weight: 300;
 }
 
-#input1 {
-    width: 15em;
-    line-height: 20px;
+.input {
+  line-height: 20px;
+  width: 15em;
 }
 
-.button {
-    background-color: var(--orange);
-    border-radius: 8px;
-    border-style: none;
-    box-sizing: border-box;
-    color: white;
-    cursor: pointer;
-    display: inline-block;
-    font-family: inherit;
-    font-size: 14px;
-    font-weight: 500;
-    height: 30px;
-    line-height: 20px;
-    list-style: none;
-    margin: 0;
-    outline: none;
-    padding: 5px 16px;
-    position: relative;
-    text-align: center;
-    text-decoration: none;
-    transition: color 100ms;
-    vertical-align: baseline;
+.button-submit {
+  background-color: var(--orange);
+  border-radius: 8px;
+  border-style: none;
+  color: var(--text);
+  cursor: pointer;
+  display: inline-block;
+  font-family: inherit;
+  font-size: 16px;
+  font-weight: 500;
+  padding: 10px 16px;
```

Comment:
Yes to the alphabetical order!!! So much easier to look up a property especially if the list is long.